doc/nrf/releases/release-notes-latest.rst

```diff
@@ -250,6 +250,10 @@ Trusted Firmware-M:
 -------------------
 
 * Added a simple sample that demonstrates how to integrate TF-M in an application.
+* Enabled using the platform code that resides outside of the Trusted Firmware-M
+  repository. This allows for providing configurable memory partitioning in the
```

Comment:
@ioannisg please change to adhere to 'one sentence - one line' rule in RST docs ( no 100 char limit here)
So the first sentence should not be broken and the second one should start in newline.